pages/feed.vue

```diff
@@ -36,27 +38,16 @@ import { useCities } from '~/use/cities'
 import { useDoc } from '~/use/doc'
 import { useApp } from '~/use/app'
 import { getUrlFromText } from '~/utils'
-import { postTypeList } from '~/use/posts'
-import TInputSelect from '~/components/TInput/TInputSelect.vue'
+import { usePosts } from '~/use/posts'
 
 export default {
-  components: { TInputSelect },
   setup() {
     const { uid, username } = useAuth()
     const { currentCity } = useCities()
     const { getPlace } = useApp()
     const newMessage = ref('')
     const postType = ref('')
-
-    const filterTypeList = [
-      { label: 'Newest', value: 'Newest' },
-      { label: 'Hot', value: 'Hot' },
-      { label: 'Popular', value: 'Popular' },
-      { label: 'Watching', value: 'Watching' },
-      { label: 'Starred', value: 'Starred' },
-      { label: 'Archived', value: 'Archived' },
-      { label: 'Yours', value: 'Authored' },
-    ]
+    const { postTypeList, filterTypeList } = usePosts()
```

Comment:
where `filterTypeList` is from?

Reply:
`usePosts()` returns object. This way you can access its properties. Read more about Object destructuring in javascript.

Reply:
Comments are not for discussions. They are to request changes. Every comment considered to be an issue and requires revision. Please use Slack for discussions.